.gitignore

```diff
@@ -17,6 +17,9 @@ pickle-email-*.html
 # TODO Comment out this rule if you are OK with secrets being uploaded to the repo
 config/initializers/secret_token.rb
 
+# Ignore Visual Studio Code files
+/.vscode
+
```

Comment:
this seems specific to personal choice of editor . . . 🤔 
what are these files?

Reply:
this is to ignore my color scheme for each folder =D

but i just copied this line from their openproject core gitignore, so it seemed fine to add here in my mind.